modules/install/pages/admin.adoc

```diff
@@ -1,24 +1,161 @@
-= Creating Administration Accounts
+= Managing JBoss EAP
 
-Brief overview of how to create admin users
+== JBoss EAP 8 Management Options
 
-== Creating Administrator Accounts
+* JBoss EAP 8 has multiple subsystems that you can customize to support Jakarta EE application requirements, such as database connectivity, batch processing, and integration tools.
 
-* Talk about the *add-user* script and other ways to create accounts
-* Distinguish between management users and application users
-* CSV file format
-* Accessing the JBoss EAP web console
-* Accessing the JBoss EAP CLI
-* Do not go too much into details about EAP CLI or web console here. Remember there is a separate chapter for EAP config...
+* The subsystem configuration is maintained in the `standalone.xml` file for the standalone mode, and the `domain.xml` file for a managed domain mode.
 
-== References
+* These configuration files can be customized using three different approaches:
 
-* Add links to product docs
+** **Management console**: As a system administrator, you can manage most of the capabilities of your standalone or managed domain deployments by using a browser.
 
-== Lab: Creating an Administrator Account
+** **Management Command Line Interface(CLI)**:
+The CLI offers a management model for viewing and modifying attributes and for performing operations, on a standalone server or a managed domain.
 
-. Step 1
-. Step 2
-.. Step 2.a
-.. Step 2.b
-. Step 3
+** **Edit XML configuration files manually**:
+The settings of a standalone server or a managed domain are maintained in XML-based configuration files, which can be modified directly.
```

Comment:
Add a WARNING that editing XML files directly should be avoided as much as possible and that either the CLI or the web console is preferred method

Reply:
Added the warning